pkg/persistence/config/writer/config_writer.go

```diff
@@ -149,7 +149,7 @@ func toTopLevelDefinitions(context *WriterContext, configs []config.Config) (map
 
 		apiCoord := apiCoordinate{
 			project: coord.Project,
-			api:     coord.Type,
+			api:     coord.extendedType,
```

Comment:
nit: maybe use the chae and rename this thing to type? and maybe the whole thing to something clearer that 'apiCoordinate' - can't think of anything though

Reply:
I'm not sure here, so I think any further change should be done in a follow up ticket